<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROIPropertiesGui</class>
 <widget class="QMainWindow" name="ROIPropertiesGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>1119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROI Properties</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tblT1">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>1161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>931</width>
      <height>0</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>Concentration (mM)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T1(ms)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>R1(1/s)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Y(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Z(mm)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T1 Array</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblT2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>360</y>
      <width>1161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>931</width>
      <height>0</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>Concentration (mM)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T1(ms)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>R1(1/s)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Y(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Z(mm)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>T2 Array</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>5</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phantom Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>5</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field (T)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtField">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>5</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtComment">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>30</y>
      <width>196</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>5</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature (C)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtTemperature">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>5</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># of ROI sets</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtnROIsets">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblT2_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>605</y>
      <width>1161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>931</width>
      <height>0</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>Concentration (mM)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T1(ms)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>R1(1/s)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Y(mm)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Z(mm)</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>615</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>PD Array</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPhantomName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>155</y>
      <width>95</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbT1ReflectX">
       <property name="text">
        <string>reflect on X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbT1ReflectY">
       <property name="text">
        <string>reflect on Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbT1ReflectZ">
       <property name="text">
        <string>reflect on Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_ROI_File"/>
    <addaction name="actionSave_ROI_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ROI_File">
   <property name="text">
    <string>Open ROI File</string>
   </property>
  </action>
  <action name="actionSave_ROI_File">
   <property name="text">
    <string>Save ROI File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
